### PR TITLE
Switch to Minitest in Adcommon

### PR DESCRIPTION
**Pull request by seuros**
WIP

### CODE REVIEW

File: ads_common/lib/ads_common/api.rb [hunk 1]
```diff
@@ -22,7 +22,6 @@
 require 'ads_common/env_check'
 
 require 'ads_common/config'
-require 'ads_common/errors'
```
Comment (mcloonan) [inline ads_common/lib/ads_common/api.rb]: Can you explain or revert this change?

File: ads_savon/Gemfile [hunk 2]
```diff
@@ -0,0 +1,2 @@
+source 'https://rubygems.org'
```
Comment (mcloonan) [inline ads_savon/Gemfile]: I don't think we're interested in using bundler at this time. Is there a particular advantage you see during development?

File: ads_common/lib/ads_common/api.rb (hunk 1 above)
Comment (seuros) [inline ads_common/lib/ads_common/api.rb]: `ads_common/config` already requires `ads_common/errors`

**Comment by mcloonan** [comment]
We have decided not to migrate to minitest at this time.
